symplyphysics/laws/kinematics/kinetic_energy_of_body_from_mass_and_velocity.py

```diff
@@ -0,0 +1,22 @@
+from symplyphysics import (
+    symbols, Eq, pretty, solve, Quantity, units,
+    validate_input, validate_output, expr_to_quantity
+)
+
+# Description
+# Kinetic energy of body EK = (m * U^2) / 2
+# where :
+# m - body mass
+# U - body velocity
+kinetic_energy_of_body, body_mass, body_velocity = symbols('kinetic_energy_of_body body_mass body_velocity')
+law = Eq(kinetic_energy_of_body, body_mass * body_velocity**2 / 2)
+
+def print():
+    return pretty(law, use_unicode=False)
+
+@validate_input(body_mass_=units.mass, body_velocity_=units.velocity)
+@validate_output(units.energy)
+def calculate_kinetic_energy_body(body_mass_: Quantity, body_velocity_: Quantity) -> Quantity:
```

Comment:
This name does not make much sense. There is no such entity as `energy_body`. Make it simply `calculate_kinetic_energy`